tools/lkl/tests/vfio-pci.c

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
it's not serious though, but can we use the different file names (tools/lkl/{lib,tests}/vfio-pci.c) to avoid confusion ?

Reply:
tools/lkl/tests/vfio-pci.c was renamed tools/lkl/tests/disk-vfio-pci.sh 
https://github.com/lkl/linux/pull/487/files#diff-82a4e2ece37edbdaa134fb4064fa82ddR1